### DISCUSSION TITLE
Analytics and dashboard 📊

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#215] [Ideas]
User needs to be able to create custom graphs and dashboards on top of records.

As a first step, we could rely on an external js library or external service in docker such as https://lightdash.com
The first path (js lib) would be preferred

**Comment by nghiahung-dev** [comment]
+1

**Comment by StephanieJoly4** [comment]
ex of data point to show (asked by a customer today: daily reporting of the # of companies and accounts daily added per Twenty user => way for managers to check team's activity)

**Comment by FelixMalfait** [comment]
Closing since dashboards are pretty good now!
